docs/install/install-cave.md

```diff
@@ -78,7 +78,16 @@ This method is recommended for beginners, or those with less computer knowledge
 
 #### System Requirements
 
-- [VMWare Workstation Player](https://www.vmware.com/products/workstation-player/workstation-player-evaluation.html) must be installed (free software)
+- [VMWare Workstation Player](https://www.vmware.com/products/workstation-player/workstation-player-evaluation.html) must be installed (free software):
+- For high definition monitors (4k), you will want to enable the high DPI setting for VMWare Workstation Player
+     1. Create a desktop shortcut for VMWare Workstation Player
+     1. Right-click the shortcut and select Properties
+     1. Open the Compatability Tab
+     1. Select the "Change high DPI settings" button
+     1. Check the "High DPI scaling ovveride" checkbox and choose "Application" in the enabled dropdown
```

Comment:
Your numbered list showed up as roman numerals instead of numbers in case you wanted to change that:
![image](https://user-images.githubusercontent.com/65614396/169162050-2c251687-d94d-444c-b412-e8e88ba39e37.png)